tests/ops/op_math/test_pow_op.py

```diff
@@ -0,0 +1,612 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from copy import copy
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.operation import DecompositionUndefinedError
+from pennylane.ops.op_math.pow_class import Pow, PowOperation
+
+
+class TestInheritanceMixins:
+    """Test inheritance structure and mixin addition through dynamic __new__ method."""
+
+    def test_plain_operator(self):
+        """Test when base directly inherits from Operator, Adjoint only inherits
+        from Adjoint and Operator."""
+
+        class Tester(qml.operation.Operator):
+            num_wires = 1
+            num_params = 1
+
+        base = Tester(1.234, wires=0)
+        op = Pow(base, 1.2)
+
+        assert isinstance(op, Pow)
+        assert isinstance(op, qml.operation.Operator)
+        assert not isinstance(op, qml.operation.Operation)
+        assert not isinstance(op, qml.operation.Observable)
+        assert not isinstance(op, PowOperation)
+
+        # checking we can call `dir` without problems
+        assert "num_params" in dir(op)
+
+    def test_operation(self):
+        """When the operation inherits from `Operation`, the `AdjointOperation` mixin should
+        be added and the Adjoint should now have Operation functionality."""
```

Comment:
No explicit Adjoint usage in this test either, is it a typo, or an implicit thing being checked?

Reply:
copy and paste error:

```suggestion
        """When the operation inherits from `Operation`, the `PowOperation` mixin should
        be added and the Pow should now have Operation functionality."""
```